app/models/MessageModel.php

```diff
@@ -50,6 +55,37 @@ function exists($id) {
       return ($this->get($id) !== NULL);
     }
 
+    function getNumUnread($studentOrRecruiterId) {
+      if (StudentModel::exists($studentOrRecruiterId)) {
+        return StudentModel::getNumUnread(new MongoId($studentOrRecruiterId));
```

Comment:
Rather than use new MongoId here, just have the parameter be of type MongoId.

Also, new functions like these should be added in the new format with the public static keyword and added at the top.